packages/app/src/components/catalog/EntityPage.test.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { ApiProvider, ApiRegistry, errorApiRef } from '@backstage/core-api';
+import { CICDSwitcher } from './EntityPage';
+import { UrlPatternDiscovery } from '@backstage/core';
+import {
+  buildKiteApiRef,
+  BuildKiteApi,
+} from '@roadiehq/backstage-plugin-buildkite';
+import { renderWithEffects, wrapInTestApp } from '@backstage/test-utils';
+
+const mockErrorApi: jest.Mocked<typeof errorApiRef.T> = {
```

Comment:
there's also a `MockErrorApi` in `test-util` if it fits your needs

Reply:
`wrapInTestApp` even provides that mock by default, so can get rid of it completely

Reply:
Thanks. Code updated.